specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/ContainerAppsJobs_ListByResourceGroup.json

```diff
@@ -0,0 +1,100 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "api-version": "2022-11-01-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/containerAppsJobs/testcontainerAppsJob0",
+            "name": "testcontainerAppsJob0",
+            "type": "Microsoft.App/containerAppsJobs",
+            "location": "East US",
+            "properties": {
+              "provisioningState": "Succeeded",
+              "managedEnvironmentId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/demokube",
+              "configuration": {
+                "activeDeadlineSeconds": 10,
+                "replicaRetryLimit": 10,
+                "manualTriggerConfig": {
+                  "completions": 4,
```

Comment:
change names for completions and replicas and make replicacount <= to parallelism